streaming/src/main/scala/org/apache/spark/streaming/util/WriteAheadLogUtils.scala

```diff
@@ -36,6 +36,7 @@ private[streaming] object WriteAheadLogUtils extends Logging {
   val DRIVER_WAL_ROLLING_INTERVAL_CONF_KEY =
     "spark.streaming.driver.writeAheadLog.rollingIntervalSecs"
   val DRIVER_WAL_MAX_FAILURES_CONF_KEY = "spark.streaming.driver.writeAheadLog.maxFailures"
+  val DRIVER_WAL_BATCHING_CONF_KEY = "spark.streaming.driver.writeAheadLog.enableBatching"
```

Comment:
maybe rename this to `spark.streaming.driver.writeAheadLog.allowBatching`, just in case we have add a config `driver.wal.enable` (similar to `receiver.wal.enable`) in future.